code/mob/living/critter/big_animal.dm

```diff
@@ -10,21 +10,32 @@
 	speechverb_say = "growls"
 	speechverb_exclaim = "roars"
 	speechverb_ask = "meows"
+	death_text = "%src% gives up the ghost!"
 	hand_count = 2
-	can_throw = 1
-	can_grab = 1
-	can_disarm = 1
+	can_throw = TRUE
+	can_grab = TRUE
+	can_disarm = TRUE
 	butcherable = 1
-	name_the_meat = 1
-	max_skins = 1
-	add_abilities = list(/datum/targetable/critter/slam,
-						/datum/targetable/critter/bite/big)
+	name_the_meat = TRUE
+	max_skins = 3
+	health_brute = 30
+	health_brute_vuln = 0.6
+	health_burn = 30
+	health_burn_vuln = 0.8
```

Comment:
The old mob critter lions were hilariously weak, but this makes them very tanky, like mini brullbars. For comparison they used to take 2 fire extinguisher hits and now take 10, which coupled with their buffed abilities will have a big impact on azones, prefabs and wraith summons.